src/plugins/home/server/services/sample_data/sample_data_installer.test.ts

```diff
@@ -158,16 +199,14 @@ describe('SampleDataInstaller', () => {
       expect(esClient.asCurrentUser.indices.create).toHaveBeenCalledTimes(1);
       expect(esClient.asCurrentUser.indices.create).toHaveBeenCalledWith({
         index: 'kibana_sample_data_test_single_data_index',
-        body: {
-          mappings: {
-            properties: {
-              someField: { type: 'keyword' },
-            },
-          },
-          settings: {
-            index: {},
+        mappings: {
```

Comment:
The use of the 'body' key has been deprecated, so moving the nested keys to the top level object.

node_modules/@elastic/elasticsearch/lib/api/typesWithBodyKey.d.ts line 8755